src/eeprom24x.rs

```diff
@@ -306,7 +306,9 @@ impl_for_page_size!(
     ["24x01", "AT24C01", 7, No, new_24x01],
     ["24x02", "AT24C02", 8, No, new_24x02],
     ["24CSx01", "24CS01", 7, Yes, new_24csx01],
-    ["24CSx02", "24CS02", 8, Yes, new_24csx02]
+    ["24CSx02", "24CS02", 8, Yes, new_24csx02],
+    ["24AAx02E48", "24AA02E48", 8, No, new_24aax02e48],
+    ["24AAx02E64", "24AA02E64", 8, No, new_24aax02e64]
```

Comment:
```suggestion
    ["24x02E48", "24AA02E48", 8, No, new_24x02e48],
    ["24x02E64", "24AA02E64", 8, No, new_24x02e64]
```
I would remove the `AA` part since that has no implications in the programming interface.